docs/DeveloperGuide.md

```diff
@@ -997,6 +1089,25 @@ testers are expected to do more *exploratory* testing.
 
 ---
 
+### Suggesting a show
+
+1. Prerequisites: None.
+2. Launch the app.
```

Comment:
I think we should standardize with the rest of the manual test cases where we didn't specify `Launch the app.`. 
Maybe we can use this instead? `2. List all shows using the list command.`